sources/HUBViewControllerImplementation.m

```diff
@@ -609,11 +610,18 @@ - (void)reloadCollectionViewWithViewModel:(id<HUBViewModel>)viewModel animated:(
     /* Performing batch updates inbetween viewDidLoad and viewDidAppear is seemingly not allowed, as it
      causes an assertion inside a private UICollectionView method. If no diff exists, fall back to
      a complete reload. */
-    if (!self.viewHasAppeared || self.lastViewModelDiff == nil) {
+    if (!self.hasPerformedInitialReload || !self.viewHasAppeared || self.lastViewModelDiff == nil) {
```

Comment:
With this, can we remove `viewHasBeenLaidOut`, or do they complement each other?